content/blog/getting-started-to-generate-dscc-api-client-libraries-for-python-using-openapi-generator.md

```diff
@@ -0,0 +1,217 @@
+
+Once the JAR file is downloaded, you can execute the following CLI at the folder where the JAR file is downloaded to display the brief information on how to use this JAR file.
+
+```shell
+~$ java -jar openapi-generator-cli.jar help
+```
+
+The output will be something like this:
+
+```typescript
+usage: openapi-generator-cli <command> [<args>]
+
+The most commonly used openapi-generator-cli commands are:
+    author        Utilities for authoring generators or customizing templates.
+    batch         Generate code in batch via external configs.
+    config-help   Config help for chosen lang
+    generate      Generate code with the specified generator.
+    help          Display help information about openapi-generator
+    list          Lists the available generators
+    meta          MetaGenerator. Generator for creating a new template set and configuration for Codegen.  The output will be based on the language you specify, and includes default templates to include.
+    validate      Validate specification
+    version       Show version information used in tooling
+
+See 'openapi-generator-cli help <command>' for more information on a specific
+command.
+```
+
+Now that the JAR file is downloaded and ready to use, lets create a Python SDK using the openAPI generator JAR file. The following command line is used for generating a Python client library using the openapi-generator.jar file.
+
+```shell
+~$ java -jar openapi-generator-cli.jar generate -i storage-api.yaml -g python -o sdks/dscc-python-sdk
+```
+
+* 'i' represents the input file, that is the OpenAPI spec which can be in the form of JSON or YAML.
+* 'generate' represents generating the code based on the specified generator
+* 'g' represents the generator/language name like Java, Go.
+* 'o' represents output directory where the client library will be generated.
+
+This Python DSCC client library can be generated in a few minutes. Below is a snippet of screen output during the generation a Python client library using the openapi-generator.jar file.
+
+![](/img/client-generation.jpg "Generating Python SDK using OpenAPI generator")
+
+The generated client library can be made available in your workstation, or it can also be uploaded to a GitHub library so that it can be made available for others to use. An example of the GitHub repository of a sample Python client library looks like this: 
+
+![](/img/python-open-api-sdk-repo.jpg "GitHub repository of Python Client library generated using OpenAPI generator")
+
+The client library comes with the following:
+
+* The code for assisting the DSCC API calls.
+* Documentation for this DSCC client library that derived from the API spec.
```

Comment:
Take out the word "that" so it just reads:
 "Documentation for this DSCC client library derived from the API spec."